### ISSUE TITLE
New page button in create page tab doesn't have a good contrast ratio

### ISSUE THREAD

**Issue by AfLosada** [AfLosada/Ghost_MISO#31]
**Describe the bug**
Every element should have a good contrast ratio. New page button in create page tab doesn't

**To Reproduce**
1. Create Page
2. See new element on the top left of the corner
4. Use Axe dev tools for accessibility to get actual information.

**Expected behavior**
Every element should have a good contrast ratio

**Screenshots**
[VIDEO](https://youtu.be/Li2hoPVZCV4)

**Environment (please complete the following information):**
 - Device: DESKTOP
 - OS: Windows 11
 - Browser: Chrome
 - Version 5.68.0
